src/types/index.ts

```diff
@@ -1216,16 +1216,16 @@ export interface ProcedureAuthorizationStatus {
 
 interface TransferRestrictionBase {
   /**
-   * array of Scope IDs that are exempted from the Restriction
+   * array of Scope/Identity IDs that are exempted from the Restriction
+   *
+   * @note if the Asset requires investor uniqueness, Scope IDs are used. Otherwise, we use Identity IDs. More on Scope IDs and investor uniqueness
+   *   {@link https://developers.polymesh.network/introduction/identity#polymesh-unique-identity-system-puis | here} and
+   *   {@link https://developers.polymesh.network/polymesh-docs/primitives/confidential-identity | here}
    */
-  exemptedScopeIds?: string[];
+  exemptedIds?: string[];
```

Comment:
Is it ok we are not supporting `Identity[]` here?

Reply:
The value could be a scopeId. Even if we check whether the asset has PUIS enabled or not, the agent could have exempted a ScopeId, invalid DID, or anything that fits the format